tests/TestDepBuildsInSource/__snapshots__/test.js.snap

```diff
@@ -56,7 +54,6 @@ export dep__etc=\"$dep__install/etc\"
 # [BuiltIns]
 export testdepoutofsourcebuildisfalse__name=\"TestDepOutOfSourceBuildIsFalse\"
 export testdepoutofsourcebuildisfalse__version=\"1.0.0\"
-export testdepoutofsourcebuildisfalse__source_root=\"$esy__sandbox\"
```

Comment:
Oh, I see I just didn't originally understand that now `esy_build__source_root` will only apply to the rootmost package. I suppose it makes sense. I don't see many use cases for a "cur" version.